hack/sync-ingresscontroller-crd.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -euo pipefail
+
+API_CRD="vendor/github.com/openshift/api/operator/v1/0000_50_ingress-operator_00-custom-resource-definition.yaml"
+INGRESS_CRD="manifests/00-custom-resource-definition.yaml"
+COPY_CMD=$(cp $API_CRD $INGRESS_CRD)
+
+if ! stat $INGRESS_CRD >/dev/null; then
+  $COPY_CMD
+elif diff $INGRESS_CRD $API_CRD; then
+  rc=$?
+  if [[ $rc != 0 ]]; then
```

Comment:
The fact that we are in the `elif` branch means that `$?` was non-zero, so `rc` and this `if` are redundant.